packages/jsii-rosetta/lib/languages/csharp.ts

```diff
@@ -70,6 +70,14 @@ interface CSharpLanguageContext {
 type CSharpRenderer = AstRenderer<CSharpLanguageContext>;
 
 export class CSharpVisitor extends DefaultVisitor<CSharpLanguageContext> {
+  /**
+   * Translation version
+   *
+   * Bump this when you change something in the implementation to invalidate
+   * existing cached translations.
+   */
+  public static readonly VERSION = '1';
```

Comment:
Food for (future) thought -- this feels very much like a good intentions thing. Could we mechanize this a bit better somehow?

Reply:
Maybe. Not sure!